NeoML/include/NeoML/Dnn/Layers/IndRnnLayer.h

```diff
@@ -115,14 +126,15 @@ class NEOML_API CIndRnnRecurrentLayer : public CBaseLayer {
 	void LearnOnce() override;
 
 private:
-	// int hiddenSize; // Number of elements in vector
+	IMathEngine::TIndRnnActivation activation; // Activation function
 	bool reverse; // If true then sequences must be processed in reversed order
 	float dropoutRate; // Dropout rate on recurrent link
-	CFloatHandleVar* dropoutMask; // Dropout mask
+	std::unique_ptr<CFloatHandleVar> dropoutMask; // Dropout mask
```

Comment:
std here?

Reply:
Why not? We already have `unique_ptr` in .cpp files in NeoML (e.g. BaseLayer.cpp)

Reply:
It is a header. We don't use std in headers...

Reply:
fixed